src/Components/Web.JS/src/Platform/Mono/MonoTypes.d.ts

```diff
@@ -29,3 +29,19 @@ declare namespace MONO {
 // mono_bind_static_method allows arbitrary JS data types to be sent over the wire. However we are
 // artifically limiting it to a subset of types that we actually use.
 declare type BoundStaticMethod = (...args: (string | number | null)[]) => (string | number | null);
+
+declare namespace BINDING {
+  function js_string_to_mono_string(jsString: string): Pointer;
+  function js_typed_array_to_array(array: Uint8Array): Pointer;
+  function js_typed_array_to_array<T>(array: Array<T>): Pointer;
+  function conv_string(dotnetString: System_String | null): string | null;
```

Comment:
This is the replacement to `toJavaScriptString`. However the mono code has the exact warnings about being unsafe that our function had. @lewing said he'd investigate their code. That said, we use this function indirectly as part of `mono_bind_static_method`, so it's not making things worse. 

Mono's bindings also has something fairly close to `toUint8Array`, but it does not guarantee an `UInt8Array`, so I'm leaving that be for now.

Reply:
```suggestion
  function conv_string(dotnetString: System_String): string | null;
```

... because `System_String` includes the .NET notion of `null`. There aren't any cases where we'd need to be passing a JavaScript `null` here, are there?